libs/base/src/lib/context/NetworkProvider.tsx

```diff
@@ -170,20 +170,23 @@ const ETH_MAINNET: EthereumMainnet = {
   gasStationEndpoint: 'https://gas.mycryptoapi.com/',
   gqlEndpoints: {
     protocol: [
-      graphHostedEndpoint('mstable', 'mstable-protocol'),
+      'https://api.studio.thegraph.com/query/32034/mstable-protocol/v0.1.2',
       graphMainnetEndpoint('0x26cf67040678eb0f5654c9cbaad78dc1694cbafa', 0, process.env.NX_PROTOCOL_SUBGRAPH_API_KEY as string),
     ],
-    staking: [graphHostedEndpoint('mstable', 'mstable-staking')],
+    staking: ['https://api.studio.thegraph.com/query/32034/mstable-staking/v0.1.0', graphHostedEndpoint('mstable', 'mstable-staking')],
     questbook: ['https://europe-west1-mstable-questbook.cloudfunctions.net/questbook'],
-    merkleDrop: [graphHostedEndpoint('mstable', 'mstable-merkle-drop')],
+    merkleDrop: [
+      'https://api.studio.thegraph.com/query/32034/mstable-ecosystem/v0.1.0',
+      graphHostedEndpoint('mstable', 'mstable-merkle-drop'),
+    ],
     snapshot: ['https://hub.snapshot.org/graphql'],
     feeders: [
       // Temporary preview URL because indexers haven't picked up the new version...
       'https://api.studio.thegraph.com/query/32034/mstable-feeder-pools-and-vaults/v0.0.13',
```

Comment:
@calvinkei did you also deploy this one?

Reply:
no since I saw this on studio already

Reply:
@calvinkei I would recommend to publish a new version and test with it